app/controllers/councillor_contributions_controller.rb

```diff
@@ -0,0 +1,38 @@
+class CouncillorContributionsController < ApplicationController
+  before_action :check_if_feature_flag_is_on
+
+  def new
+    @authority = Authority.find_by_short_name_encoded!(params[:authority_id])
+
+    if params["councillor_contribution"]
+      @councillor_contribution = @authority.councillor_contributions.build(councillor_contribution_params)
+    else
+      @councillor_contribution = CouncillorContribution.new
+    end
+
+    @councillor_contribution.suggested_councillors.build({email: nil, name: nil})
+  end
+
+  def create
+    @authority = Authority.find_by_short_name_encoded!(params[:authority_id])
+    @councillor_contribution = @authority.councillor_contributions.build(councillor_contribution_params)
+
+    if @councillor_contribution.save
+      redirect_to new_contributor_url(councillor_contribution_id: @councillor_contribution.id)
+    else
+      render :new
+    end
+  end
+
+private
```

Comment:
This should be indented to the level of the methods it applies to https://github.com/bbatsov/ruby-style-guide#indent-public-private-protected